public/javascripts/showSongsIn.js

```diff
@@ -0,0 +1,66 @@
+var showSongsIn = {
+    view: function(ctrl, args) {
+        return m('div', [
+            m('h5', 'Show songs in: ', [
+                m('.btn-group', [
+                    m('button.btn.btn-default.dropdown-toggle[type=button]', {
+                        'data-toggle': "dropdown",
+                        'aria-haspopup': "true",
+                        'aria-expanded': "false"
+                    }, (args.langShown() === 'All' ? 'All Languages' : args.langShown()), [
+                        m('span.caret')
+                    ]),
+                    m('ul.dropdown-menu', [
+                        m('li#All', [
```

Comment:
I don't think you should use these ids here. I also noticed in an earlier file that languages were hardcoded as element ids. I think they should be dynamic and not have to depend on element ids if possible.